person.rb

```diff
@@ -0,0 +1,25 @@
+# Person class
+class Person
+  def initialize(id, age, name = 'unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  # Getter methods...
```

Comment:
- [ ] [REQUIRED] Good job so far! Previous reviewer said that your `Person.rb` should be created `:name` and `:age` instance variables are the only ones that should be able to be modified that's why they are defined as `attr_accessors` (here both Getters and Setters are defined) and `:id` should only be able to be retrieved so only the `Getter` is defined 🙏🏼 
```ruby
  attr_accessor :name, :age 
  attr_reader :id 
```